napari/utils/settings/_manager.py

```diff
@@ -47,8 +47,6 @@ class SettingsManager:
     _FILENAME = "settings.yaml"
     _APPNAME = "Napari"
     _APPAUTHOR = "Napari"
-    application: ApplicationSettings
-    plugin: PluginSettings
```

Comment:
I'm -1 on this removal.  the problem with this is that it makes for a really bad dev experience in an IDE. With these type hints, you get tab completion, mypy checks, linting, etc... without them, we have to go into `settings/_defaults.py` each time to remember what the name is, what type they are etc...  With this pattern we lose a lot of the really nice parts of having pydantic settings.  What's the motivation for removing?

Reply:
I can add them back. No problem.

Reply:
before this change, in an IDE:

<img width="789" alt="Untitled 2" src="https://user-images.githubusercontent.com/1609449/112066192-495c0c80-8b3c-11eb-86ac-ec6a01e273ed.png">
<img width="774" alt="Untitled 3" src="https://user-images.githubusercontent.com/1609449/112066196-4bbe6680-8b3c-11eb-9d4f-0670ddc08e99.png">

after this change:
<img width="562" alt="Untitled" src="https://user-images.githubusercontent.com/1609449/112066220-5678fb80-8b3c-11eb-84c6-bbb739c13e0a.png">

Reply:
thanks!

Reply:
Fixed!